### DISCUSSION TITLE
Quick question on creating custom model

### DISCUSSION THREAD

**Discussion by medledan** [TheLastBen/fast-stable-diffusion#1208] [Q&A]
I have about 25 instance images of my subject, and about 1,000 concept images I am using to place my subject within a specific body type. This appears to be working all right but I did just read somewhere that if I am just training my model for a specific face I don't even need the concept images. I am using the SD 1.5 model and thinking about using the F222 next. So would these models have more than enough body diversity for me where I don't even need the concept images? \

Thank you for your time.

**Comment by TheLastBen** [comment]
try without concept images, if you don't get enough diversity, then resume the training on only the concept images by settings the unet and text_enc steps to 0
